makefile

```diff
@@ -1,4 +1,5 @@
 VERSION  := 5.2.0
```

Comment:
I'm a little rusty on make-isms, but I think this needs to be  `VERSION ?= 5.2.0` so that `VERSION` can be overridden with command line arguments